<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>del_teachers</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 255, 255);</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>431</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Имя</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Отчество</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="combo_SNP">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ФИО</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_del_teacher">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>321</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>281</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/pngwing.com (8).png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
